thirdeye/thirdeye-frontend/app/pods/self-serve/create-alert/controller.js

```diff
@@ -448,10 +440,21 @@ export default Ember.Controller.extend({
         }
       };
 
+      // Set granularity types
       if (granularity.includes('minute') || granularity.includes('5-minute')) { gkey = 'minute'; }
       if (granularity.includes('hour')) { gkey = 'hour'; }
       if (granularity.includes('day')) { gkey = 'day'; }
 
+      // Add filter and dimension choices if available
+      if (gkey !== 'minute') {
+        if (Ember.isPresent(selectedFilter)) {
+          settingsByGranularity.common.filters = selectedFilter;
```

Comment:
As discussed, we should double check the expected format of the filters.
`{"continent": ["US", "CN"]}` or `{"continent": ["US"],"continent": ["CN"]}`